### PR TITLE
c8d: ImageService.softImageDelete: use OCI and containerd constants

### PR DESCRIPTION
**Pull request by thaJeztah**
relates to https://github.com/moby/moby/pull/44598#discussion_r1203565591

**- What I did**

**- How I did it**

**- How to verify it**

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: daemon/containerd/soft_delete.go [hunk 1]
```diff
@@ -32,8 +33,8 @@ func (i *ImageService) softImageDelete(ctx context.Context, img containerdimages
 		danglingImage := img
 
 		danglingImage.Name = danglingImageName(img.Target.Digest)
-		delete(danglingImage.Labels, "io.containerd.image.name")
-		delete(danglingImage.Labels, "org.opencontainers.image.ref.name")
+		delete(danglingImage.Labels, containerdimages.AnnotationImageName)
+		delete(danglingImage.Labels, ocispec.AnnotationBaseImageName)
```
Comment (thaJeztah) [inline daemon/containerd/soft_delete.go:37]: oh! wait; picked the wrong one; fixing now

Comment (thaJeztah) [reply daemon/containerd/soft_delete.go]: done

**Review by laurazard — APPROVED** [review]
LGTM

**Review by vvoland — APPROVED** [review]

**Comment by thaJeztah** [comment]
thx both! Let me bring this one in

**Pull request merged**